src/platforms/common/configuration/options.mdx

```diff
@@ -775,6 +775,17 @@ If <PlatformIdentifier name="trace-propagation-targets" /> is not provided, trac
 
 </ConfigKey>
 
+<ConfigKey name="functions-to-trace" supported={["python"]}>
+
+An optional list of functions that should be set up for performance monitoring. For each function in the list a span will be created when the function is executed.
+Functions in the list are represented as strings containing the fully qualified name of the fuction.
+
+This is a convenience option to not have manual instrumentation scattered all over your code base, but have one central place to configure what functions to trace.
```

Comment:
```suggestion
This is a convenient option, making it possible to have one central place for configuring what functions to trace, instead of having manual instrumentation scattered all over your code base.
```